### PR TITLE
refactoring structure of the project to improve unit tests

### PR DESCRIPTION
**Pull request by geonexus**
### REVIEWERS
@flopezag @jframos 

### DESCRIPTION
- server file moved to facts package.
- new unit tests to test server.
- Improved the way to execute server and tests executions

### CODE REVIEW

File: tests/test_facts.py [hunk 1]
```diff
@@ -24,20 +24,60 @@
...
+                and query.__contains__("WHERE tenantId=\"tenantId\""):
+            self.result = [["tenantId", 5]]
+
+    def fetchall(self):
+        # returns the result of the query
+        return self.result
+
+
+class MyAppTest(unittest.TestCase):
+
```
Comment (flopezag) [inline tests/test_facts.py]: What is the purpose of this class, please introduce a description of the class.

Comment (geonexus) [inline tests/test_facts.py]: fixed

**Comment by flopezag** [comment]
LGTM

File: tests/test_facts.py [hunk 2]
```diff
@@ -24,20 +24,57 @@
 
 from flask.ext.testing import TestCase
 from flask import Flask
+from mockito import *
```
Comment (jframos) [inline tests/test_facts.py]: Do you need import the whole mockito lib?

**Comment by jframos** [comment]
After this PR, we must update the Jenkins' jobs.

**Comment by jframos** [comment]
LGTM

**Pull request merged**